lib/msal-browser/src/error/BrowserAuthErrorCodes.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const pkceNotGenerated = "pkce_not_created";
```

Comment:
Will all these minify into the array they are referred in above? I thought minification cannot work when the `constants` scope cannot be defined. Since the usage is spread across the library, was curious how this helps.

Reply:
It's a bit complex, I'll try to break it down (I also plan to do a more in depth demonstration for the team at a later date)

Each of these will be minified down to something like `a="pkce_not_created"` and any internal reference to this constant will instead reference `a`.

It also means that any unused error codes can be removed because they are all individual constants rather than being lumped together in some shared object.

We also ` export * as BrowserAuthErrorCodes` so that consumers of the library can do something like `BrowserAuthErrorCodes.pkceNotGenerated`. This gets included in our output bundle separately as something along the lines of `BrowserAuthErrorCodes={pkceNotGenerated: a, ...etc}`

The final bundler (application) can then map references to `BrowserAuthErrorCodes.pkceNotGenerated` in the app back to `a` and remove the `BrowserAuthErrorCodes` object from the final bundle.

Reply:
Thanks. Good improvement over errors and probably adds to better docs.